flink-runtime/src/main/java/org/apache/flink/runtime/asyncprocessing/EpochManager.java

```diff
@@ -68,12 +68,20 @@ public enum ParallelMode {
     /** Current active epoch, only one active epoch at the same time. */
     Epoch activeEpoch;
 
+    /** The epoch that will be provided to new records. */
+    @Nullable Epoch overrideEpoch;
```

Comment:
IIUC, `overrideEpoch`  is a `closed` epoch which prepares for finishing.
How about renaming it to `finishingEpoch`?

Reply:
Renamed accordingly